descriptors/ModuleDescriptor-template.json

```diff
@@ -223,7 +223,8 @@
             "inventory-storage.instances.item.post",
             "inventory-storage.instances.item.put",
             "inventory-storage.preceding-succeeding-titles.item.post",
-            "configuration.entries.collection.get"
+            "configuration.entries.collection.get",
+            "source-storage.verified.records"
```

Comment:
Do we need to add the related interface to `requires` section?

Reply:
Added